### PR TITLE
[OpsGenie] get alert logs

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
added 3 commands

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml [hunk 1]
```diff
@@ -1286,6 +1288,38 @@ script:
       description: Notify name of the routing rule.
     - contextPath: OpsGenie.TeamRoutingRule.notify.type
       description: Notify type of the routing rule.
+  - name: opsgenie-get-alert-logs
+    arguments:
+    - name: alert_id
+      required: true
+      description: Alert ID
+    description: Lists alert logs.
+  - name: opsgenie-add-alert-note
```
Comment (mmhw) [inline Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml:1297]: Please add the outputs

File: Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml [hunk 2]
```diff
@@ -1286,6 +1288,38 @@ script:
...
+    - name: note
+      required: true
+      description: Alert note to add.
+    - name: user
+      required: false
+      description: Display name of the request owner.
+    - name: source
+      required: false
+      description: Display name of the request source.
+  - name: opsgenie-add-alert-details
```
Comment (mmhw) [inline Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml:1311]: Please add the outputs

**Review by mmhw — COMMENTED** [review]
Hi @ennozdd ,
Thank you for your contribution!
Good work :)

Please note my comments.

**Comment by ennozdd** [comment]
@mmhw done

**Review by mmhw — APPROVED** [review]
Good work!

**Pull request merged**